.github/workflows/tests-js.yml

```diff
@@ -0,0 +1,36 @@
+name: 'JS Tests'
```

Comment:
I've been able to get JS tests running on tric, see https://github.com/moderntribe/events-virtual/blob/master/.github/workflows/tests-js.yml.

Reply:
I was thinking about doing that with `tric`, but I wanted to see if I could barebones it. I'm good with switching it to tric too.

Reply:
Updated